### DISCUSSION TITLE
Osu!std mirror mod rankability

### DISCUSSION THREAD

**Discussion by Fabiano1337** [ppy/osu#27913] [Q&A]
Hey,
are there any plans on ranking the mirror mod for the standard gamemode ?
A lot of people are already flipping their screens and tablet drivers in osu!stable and i dont really see anyone gaining an advantage flipping their screen.

**Comment by TextAdventurer12** [comment]
Mirror is not rankable because it needs new difficulty attributes (stack leniency means distance changes compared to no mod), so it won't be rankable until real time calculation is enabled

**Comment by mixalisnew** [comment]
i might be wtong but i feel that mirror mod should not change the dif of the map or ean more or less pp or score. It should be an accessory for mind block or just preference
